content/en/docs/quickstart.md

```diff
@@ -14,15 +14,15 @@ Before you begin, you need:
 
 ## Create an OCI-compatible registry
 
-**WARNING:** The following example creates a registry with [oras-project/registry](https://github.com/oras-project/distribution/pkgs/container/registry). This registry should only be used for development purposes. When using other registries, ensure the registry is compatible with OCI Image specification v1.1.0. Starting with `v1.0.0-rc.1` of `notation`, by default, signatures are stored using [OCI Artifact Manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/artifact.md), which is defined in [OCI Image spec v1.1.0](https://github.com/opencontainers/image-spec/tree/v1.1.0-rc2)).
+> **Warning:** The following example creates a registry with [oras-project/registry](https://github.com/oras-project/distribution/pkgs/container/registry). This registry should only be used for testing or development purposes. Refer to [registries are compatible with Notary]({{< ref "/docs/faq#what-registries-are-compatible-with-notary" >}}) for more alternatives. See [Authenticate with OCI-compliant registries]({{< ref "/docs/how-to/registry-authentication" >}}) when you log in to another OCI registry. 
 
 Create and run an OCI-compatible registry on your development computer using Docker and the [oras-project/registry](https://github.com/oras-project/distribution/pkgs/container/registry) container image. The following command creates a registry that is accessible at `localhost:5000`.
 
 ```console
 docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.4
 ```
 
-**NOTE:** For Apple silicon, add the `--platform linux/arm64` parameter.
+> **Note:** For Apple silicon, add the `--platform linux/arm64` parameter.
```

Comment:
I think we can remove this note as docker can detect the platform automatically.

Reply:
@shizhMSFT @FeynmanZhou please don't remove unless you can test and confirm this is the case for apple silicon macs

Reply:
@zr-msft @shizhMSFT Could you help to confirm this on an Apple silicon Mac? I don't have that device to verify it.

Reply:
I've tested on an M1 mac and it does not require the `--platform linux/arm64` parameter.

One thing interesting that the _Control Center_ app on mac listens to port 5000 for the _AirPlay Receiver_ feature. Therefore, running `docker run -d -p 5000:5000 registry` will fail due to port binding failure.

Reply:
@shizhMSFT Thanks for the verification, I see. I can raise another PR to update the note.